Week-3/Week-3/CallBindApply/index.txt

```diff
@@ -0,0 +1,108 @@
+Bind:-
```

Comment:
Any reason why this file is in a .txt format? Let's use `.js` extention for all the assignments and `.md` for any explanation.